src/main/java/com/openmrs/migrator/core/utilities/ConsoleUtils.java

```diff
@@ -0,0 +1,117 @@
+package com.openmrs.migrator.core.utilities;
+
+import java.io.Console;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class ConsoleUtils {
+
+  public static Optional<String> getDatabaseDetaName(Console console) {
+
+    checkConsoleAvailability(console);
+    Optional<String> wrappedName = Optional.empty();
+
+    console.writer().println("Database name:");
+    String dataBaseName = console.readLine();
+
+    wrappedName = Optional.of(dataBaseName);
+
+    return wrappedName;
+  }
+
+  private static void checkConsoleAvailability(Console console) {
```

Comment:
This could do with two unit tests (true & false)

Reply:
also best to place private functions below public ones

Reply:
I agree, But as I mentioned  the `System.console()`  returns  null  in the IDE